cns/service/main.go

```diff
@@ -891,13 +901,26 @@ func InitializeCRDState(ctx context.Context, httpRestService cns.HTTPService, cn
 	}
 	httpRestServiceImplementation.SetNodeOrchestrator(&orchestrator)
 
-	// build default clientset.
-	kubeConfig, err := ctrl.GetConfig()
-	kubeConfig.UserAgent = fmt.Sprintf("azure-cns-%s", version)
-	if err != nil {
-		logger.Errorf("[Azure CNS] Failed to get kubeconfig for request controller: %v", err)
-		return err
+	// Create the kubernetes client
+	var (
+		kubeConfig *rest.Config
+		err        error
+	)
```

Comment:
this takes up two more precious vertical lines than 
```suggestion
	var kubeConfig *rest.Config
	var err error
```

😢

Reply:
fixed